source/_docs/automation/services.markdown

```diff
@@ -0,0 +1,43 @@
+---
+title: "Automation Services"
+description: "How to use the various automation services."
+---
+
```

Comment:
This part of the documentation now feels very out of place.

I think it needs at least a couple of lines to indicate what is on this page.